js-foundations.js

```diff
@@ -3,23 +3,73 @@
 
 // 1. Write an arrow function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
 
+//  let divis = (num1,num2,num3, num4) => {
+//      if(num1, num2, num3, num4 % 3===0){
+//          return num1,num2,num3,num4, "is divisble by three"
+//      } else{
+//          return "is not divisible by three"
+//      }
+// console.log(divis(15, 0, 385, -7))
+
+let divis = (num1) => {
+     if(num1 % 3===0){
+         return num1 + " is divisble by three"
+     } else{
+         return num1 + " is not divisible by three"
+     }
+}
+console.log(divis(15))
+console.log(divis(-7))
+console.log(divis(385))
+console.log(divis(0))
+
 
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
 
+var helloMe= {
+  firstName: "Adam",
+  lastName: "Cruse",
+  Job: "Electrician",
+  yearBorn: 1994,
+}
+
+let {firstName,lastName,job,yearBorn} = helloMe
+var adam = `Hello ${helloMe.firstName} ${helloMe.lastName}, you work as an ${helloMe.Job}, and you were born in ${helloMe.yearBorn}.`
```

Comment:
Take a look at this below. It should help you get on the right track.
```
var person = {
  name: "Alex Keaton",
  phone: 123456789,
  getData: function() { return this.name+"-"+this.phone }
};

person.getData();   -> "Alex Keaton-123456789"
```
https://github.com/learn-academy-2019-delta/Syllabus/blob/master/js_beginning/07js_objects.md